### ISSUE TITLE
different  examples

### ISSUE THREAD

**Issue by sonfire186** [4uiiurz1/keras-arcface#22]
I see different examples in README.md and archs.py 

```
x = Dense(args.num_features, kernel_initializer='he_normal',
                kernel_regularizer=regularizers.l2(weight_decay))(x)
x = BatchNormalization()(x)
output = ArcFace(10, regularizer=regularizers.l2(weight_decay))([x, y])
```

```
x = Dense(512, kernel_initializer='he_normal')(x)
x = BatchNormalization()(x)
output = ArcFace(num_classes=10)([x, y])
```

Which examples is correct?

**Comment by rlrahulkanojia** [comment]
The first one seems better as it includes weight decay regularization, but both are correct.
